quickwit-indexing/src/actors/merge_executor.rs

```diff
@@ -354,27 +377,22 @@ impl MergeExecutor {
             replaced_segments_demux_field_readers,
             demuxed_virtual_splits,
         );
-        info!("demux-build-mapping-end");
```

Comment:
you can remove this one yes, can you also remove `info!("demux-build-mapping-start")` to be consistent, I added these when I was trying to understand which demux part was taking time.

Reply:
I did not remove it on purpose actually. Thanks for spotting!